javascript/chubbyjs/package.json

```
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "@chubbyjs/chubbyjs-framework": "^1.0.0",
+    "@chubbyjs/chubbyjs-framework-router-path-to-regexp": "^1.0.0",
+    "@chubbyjs/chubbyjs-http-message": "^1.1.1",
+    "@chubbyjs/chubbyjs-node-psr-http-message-bridge": "^1.1.0",
```

better us patch level versioning. I mean using `^` could lead to minor upgrade, then version (in conf) will be out-of-sync
```suggestion
    "@chubbyjs/chubbyjs-framework": "~1.0.0",
    "@chubbyjs/chubbyjs-framework-router-path-to-regexp": "~1.0.0",
    "@chubbyjs/chubbyjs-http-message": "~1.1.1",
    "@chubbyjs/chubbyjs-node-psr-http-message-bridge": "~1.1.0",
```

but something more, do we need to specify a version for each component, I mean, there is no component (like a meta-package) that drive all version (in order to avoid letting responsibility of version management to developer) ?

---

no there isn't cause everything is replaceable (its all written against interfaces)

---

ok, then. but please use `~` instead of `^`